backend/views/syson-diagram-interconnection-view/src/main/java/org/eclipse/syson/diagram/interconnection/view/services/ChildUsageCreationNodeToolsProvider.java

```diff
@@ -0,0 +1,43 @@
+/*******************************************************************************
+ * Copyright (c) 2024 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+
+package org.eclipse.syson.diagram.interconnection.view.services;
+
+import java.util.List;
+
+import org.eclipse.sirius.components.view.builder.IViewDiagramElementFinder;
+import org.eclipse.sirius.components.view.diagram.NodeTool;
+import org.eclipse.syson.diagram.interconnection.view.tools.PartUsageBindingConnectorAsUsageNodeToolProvider;
+import org.eclipse.syson.sysml.Element;
+import org.eclipse.syson.sysml.PartUsage;
+import org.eclipse.syson.sysml.SysmlPackage;
+import org.eclipse.syson.util.IDescriptionNameGenerator;
+
+/**
+ * Child creation node tools provider.
+ * The creation node tools provided by this class should only be added to nested child node descriptions (not first level one).
+ *
+ * @author Jerome Gout
+ */
+public class ChildUsageCreationNodeToolsProvider implements ILeveledCreationNodeToolsProvider {
+    @Override
+    public List<NodeTool> getNodeTools(Element element, IDescriptionNameGenerator descriptionNameGenerator, IViewDiagramElementFinder cache) {
+        List<NodeTool> result = List.of();
+        if (element instanceof PartUsage) {
+            var nodeDescriptionName = descriptionNameGenerator.getNodeName(SysmlPackage.eINSTANCE.getPartUsage());
```

Comment:
Why do you use `descriptionNameGenerator.getNodeName(...)` instead of `var childPartUsageNodeDescriptionName = leveledNodeDescriptionNameGenerator.getChildNodeName(element.eClass());`?
May be it's the right thing to do but I'm don't see why.

Reply:
I have used the same name as it is set in the `ChildUsageNodeDescriptionProvider` [here](https://github.com/eclipse-syson/syson/blob/9e49a6f78f2f5e61b401e9bd5429cab076befe82/backend/views/syson-diagram-interconnection-view/src/main/java/org/eclipse/syson/diagram/interconnection/view/nodes/ChildUsageNodeDescriptionProvider.java#L80) 
I did not see why the `IVDescriptionNameGenerator.getChildNodeName()` method is not used in this case, but I did not want to introduce regression if I missed something.

Reply:
Ok I see, thanks for the explanation